fuzz/fuzz_targets/parse.rs

```diff
@@ -0,0 +1,7 @@
+#![no_main]
+#[macro_use] extern crate libfuzzer_sys;
+extern crate goblin;
+
+fuzz_target!(|data: &[u8]| {
+    let _ = goblin::parse(data);
```

Comment:
Yes, I have tried this, and it works. It finds e_phoff overflow I already reported, and also very similar e_shoff overflow.